.gitlab-ci.yml

```diff
@@ -1,224 +0,0 @@
-# GitLab build configuration
```

Comment:
Might we want to keep this file around, at least until we have the GitHub version known to be working correctly?
(I realize we can always get it out of the history, but...)

Reply:
It either works or needs to be fixed.   Not need to keep clutter around, imo.